src/patternfly/components/Button/button.scss

```diff
@@ -23,7 +23,7 @@
   --#{$button}--TextDecorationColor: currentcolor;
   --#{$button}--TransitionDuration: var(--pf-t--global--motion--duration--fade--default);
   --#{$button}--TransitionTimingFunction: var(--pf-t--global--motion--timing-function--default);
```

Comment:
can we adjust this token to use `--pf-t--global--motion--timing-function--accelerate` instead? reviewed with the design team today, and thought this adjustment made the ripple look smoother! thank you!